Marlin/src/feature/bedlevel/ubl/ubl.cpp

```diff
@@ -214,7 +214,7 @@
         else if (isnan(f))
           serialprintPGM(human ? PSTR("  .   ") : PSTR("NAN"));
         else if (human || csv) {
-          if (human && f >= 0.0) SERIAL_CHAR(f > 0 ? '+' : ' ');  // Space for positive ('-' for negative)
+          if (human && f >= 0.0) SERIAL_CHAR(f > 0 ? '+' : ' ');  // Be sure sign is displayed for positive numbers only 0 miss sign (display space)
```

Comment:
Maybe something simpler?
e.g.
`Display sign for positive numbers (' ' for 0)`

Reply:
Yep, it could be, that was just something I think code wants to do.
Better message may of course be written by people whom English is their native language

Reply:
What about new one?

Reply:
I can live with that.

Reply:
I prefer to specify that also positive have sign because negative sign is automatically written by function below